src/main/java/nl/novi/TechItEasy/models/Television.java

```diff
@@ -0,0 +1,185 @@
+package nl.novi.TechItEasy.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "televisions")
+public class Television {
+    @Id
+    @GeneratedValue
+    private int id;
+
+    private String type;
+    private String brand;
+    private String name;
+    private double price;
+    private double availableSize;
+    private double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private boolean smartTv;
+    private boolean wifi;
+    private boolean voiceControl;
+    private boolean hdr;
+    private boolean bluetooth;
+    private boolean ambiLight;
+    private int originalStock;
+    private int sold;
+
+    public Television(String type, String brand, String name, double price, double availableSize, double refreshRate, String screenType, String screenQuality, boolean smartTv, boolean wifi, boolean voiceControl, boolean hdr, boolean bluetooth, boolean ambiLight, int originalStock, int sold) {
```

Comment:
Volgens de uitwerking hoef je geen constructor aan te maken, vorige week kreeg ik als feedback dat het aanmaken van een constructor wel best practice is. Dus goed dat je ze wel aangemaakt hebt.